portal.go

```diff
@@ -1754,6 +1754,43 @@ func (portal *Portal) GetMatrixUsers() ([]id.UserID, error) {
 	return users, nil
 }
 
+// TombstoneOrReIDIfNeeded returns true if the portal metadata should be synchronized
+func (portal *Portal) TombstoneOrReIDIfNeeded() (retargeted bool, tombstoned bool) {
+	if portal.Identifier.Service == "SMS" && portal.bridge.Config.IMessage.TombstoneOldRooms {
```

Comment:
This if would probably be nicer if it was inverted and returned false instead of wrapping the whole function